src/content/devlogs/vlad/march-15th-look-alive-sunshine.md

```diff
@@ -0,0 +1,21 @@
+---
+title: look-alive-sunshine

```

Comment:
```suggestion
title: look alive sunshine
```
i assume you want the display title here to have spaces, the `-`, it won't be used as the slug (filename is slug)